### PR TITLE
Update text and screens (EXPOSUREAPP-14354)

### PR DESCRIPTION
**Pull request by schauersbergern**
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14354)

### CODE REVIEW

**Review by KathrinLuecke — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/res/values-de/srs_submission_strings.xml [hunk 1]
```diff
@@ -36,33 +36,25 @@
     <!-- XHED: Submission consent disclaimer sub headline -->
     <string name="srs_submission_consent_disclaimer_subheadline_text">Bevor Sie andere warnen können, ist Ihr Einverständnis erforderlich.</string>
     <!-- XTXT: Submission consent section warn consent -->
-    <string name="srs_section_warn_consent_text">Sie bestätigen hiermit, dass Sie persönlich durch einen Selbsttest oder von einer Teststelle positiv getestet wurden. Sie können hier nicht in Vertretung von Familienmitgliedern warnen.</string>
+    <string name="srs_section_warn_consent_text">Wenn Sie positiv getestet wurden, können Sie Ihre Mitmenschen über die App warnen.\nDie Warnung nach einem positiven Selbsttest oder mit einem Testergebnis, das nicht in der App vorliegt, erfordert die Prüfung der Echtheit der App.</string>
```
Comment (Ein-Tim) [inline Corona-Warn-App/src/main/res/values-de/srs_submission_strings.xml:39]:
```suggestion
    <string name="srs_section_warn_consent_text">Wenn Sie positiv getestet wurden, können Sie Ihre Mitmenschen über die App warnen.\nDie Warnung nach einem positiven Selbsttest oder mit einem Testergebnis, das nicht in der App vorliegt, erfordert die automatische Prüfung der Echtheit der App.</string>
```

I'd suggest to add "automatic" here, to mane clear that there is no manual check involved.

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
One request for change @KathrinLuecke.

**Review by KathrinLuecke — APPROVED** [review]
UA approved

File: Corona-Warn-App/src/main/res/values-de/srs_submission_strings.xml (hunk 1 above)
Comment (schauersbergern) [reply Corona-Warn-App/src/main/res/values-de/srs_submission_strings.xml:39]: Hello @Ein-Tim we will consider your comment for the next release candidate

Comment (KathrinLuecke) [reply Corona-Warn-App/src/main/res/values-de/srs_submission_strings.xml:39]: @Ein-Tim I will check with the other colleagues who elaborated on this text whether "automatic" should be added.

**Review by SamuraiKek — APPROVED** [review]

**Pull request merged**